Storage/SQL/RemoteTabsTable.swift

```diff
@@ -115,11 +118,18 @@ class RemoteTabsTable<T>: GenericTable<RemoteTab> {
 
     override var factory: ((row: SDRow) -> RemoteTab)? {
         return { row -> RemoteTab in
+            let history: [NSURL]
+            if let historyString = row["history"] as? String {
+                let historyURLs = split(historyString ?? "") { $0 == "," }
+                history = historyURLs.filter { $0 != nil }.map { NSURL(string: $0)! }
```

Comment:
Let's be safer here. Parsing the URL can fail, and it will if we screw up and split on commas incorrectly!